### ISSUE TITLE
[Feature Request] Pushover Notification

### ISSUE THREAD

**Issue by AIR495** [henrywhitaker3/Speedtest-Tracker#603]
Can you please add notifications to Pushover.

**Comment by patrickstigler** [comment]
Yes that would be awesome, or maybe an API for HomeAssistant integration.

**Comment by henrywhitaker3** [comment]
Need to abstract the notifications stuff anyway, so when I get round to that (no promises on when) I'll add it in then.

Re API @patrickstigler there is an API currently, just not documented anywhere. If you run `php /config/www/artisan route:list` in the container it should show you all the available endpoints

**Comment by patrickstigler** [comment]
Ok sounds good, thank you. 
And I guess I take a look at the API. Thanks for the hint.
